e2e-tests/tests/basic.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { test, expect } from '@playwright/test';
+import * as uuid from 'uuid';
+import ExperimentPage from '../page-objects/main/ExperimentPage';
+import AdminPage from '../page-objects/admin/AdminPage';
+import NoExperimentsElement from '../page-objects/main/NoExperimentsElement';
+import EmpiricaTestFactory from '../setup/EmpiricaTestFactory';
+import { GamesTypeTreatment } from '../page-objects/admin/BatchesPage';
+
+const baseUrl = 'http://localhost:3000';
+
+const testFactory = new EmpiricaTestFactory({
+  installMode: 'NPM' // TODO: implement caching of the empirica project
+});
+
+test.beforeAll(async () => {
+  await testFactory.init();
+});
+
+test.afterAll(async () => {
+  await testFactory.teardown();
+});
+
+
+test.describe('Empirica', () => {
+
+  test('Empty experiemnt page loads successfully', async ({ page }) => {
+    const experimentPage = new ExperimentPage({
+      page,
+      baseUrl
+    })
+
+    await experimentPage.open();
+
+    const noExperimentsElement = new NoExperimentsElement({ page });
+
+    await expect(await noExperimentsElement.getElement()).toBeVisible();
+
+  });
+
+  test('Admin page loads successfully', async ({ page }) => {
+    const adminPage = new AdminPage({
+      page,
+      baseUrl
+    })
+
+    await adminPage.open();
+
+  });
+
+  test.only('creates batch with 1 game with one player, into view, player passes through the game', async ({ page }) => {
+    const adminPage = new AdminPage({
+      page,
+      baseUrl
+    });
+
+
+    const playerId = `player-${uuid.v4()}`;
+    const gamesCount = 1;
+    const gameMode = GamesTypeTreatment.Solo;
+    const playerAge = 25;
+    const playerGender = 'male';
+    const jellyBeansCount = 1200;
+
+    await adminPage.open();
+
+    const batchesPage = adminPage.getBatchesPage();
+
+    await batchesPage.open();
+
+    await batchesPage.createBatch({ 
+      mode: gameMode, 
+      gamesCount 
+    });
+
+    await batchesPage.startGame();
```

Comment:
I haven't looked at what createBatch does here, but just thinking, from an API standpoint, maybe more something like:

```suggestion
    const batch = await batchesPage.createBatch({ mode: gameMode, gamesCount });

    await batch.start();
```

Since there can be multiple batches listed, keeping a reference to a batch is convenient.

This is not a reason to hold up this PR, just an idea for future.

Reply:
Yes, I agree. I think some of those page objects might need to be adapted a bit as long as I have better understanding of things

Reply:
I will revisit this after I add more test ids. Currently, getting a specifc batch configuration is not that convienient